src/react/projects/spark-react/src/SprkMasthead/SprkMasthead.js

```diff
@@ -48,6 +66,61 @@ class SprkMasthead extends Component {
     }
   }
 
+  closeNarrowNavMenu() {
+    this.setState({ narrowNavOpen: false });
+  }
+
+  setIsScrolled() {
+    if (window.scrollY >= 10) {
+      this.setState({ isScrolled: true });
+    } else {
+      this.setState({ isScrolled: false });
+    }
+  }
+
+  getCurrentLayout() {
+    this.setState({ currentLayout : isElementVisible('.sprk-c-Masthead__menu')});
+  }
+
+  checkIfNarrowLayout() {
+    if (this.state.isNarrowLayout !== this.state.currentLayout) {
+      this.setState({ isNarrowLayout : this.state.currentLayout });
+      this.toggleScrollEvent();
+      if(!this.state.isNarrowLayout) {
+        this.setState({ isHidden : false });
+      }
+    }
+  }
+
+  checkLayoutOnResize() {
+    this.getCurrentLayout();
+    this.checkIfNarrowLayout();
+  }
+
+  toggleMenu() {
+    if(this.state.scrollDirection === 'down') {
```

Comment:
Could you clean up some of these `if() statements.
```suggestion
    if (this.state.scrollDirection === 'down') {
```

Reply:
Yup! Just pushed the changes.